assets/javascripts/discourse/widgets/discourse-reactions-list-emoji.js.es6

```diff
@@ -26,44 +28,10 @@ export default createWidget("discourse-reactions-list-emoji", {
     }
   },
 
-  _setupPopper(popper, selector) {
-    next(() => {
-      let popperElement;
-      const trigger = document.querySelector(`#${this.buildId(this.attrs)}`);
-
-      popperElement = document.querySelector(
-        `#${this.buildId(this.attrs)} ${selector}`
-      );
-
-      if (popperElement) {
-        if (this[popper]) {
-          return;
-        }
-
-        this[popper] = this._applyPopper(trigger, popperElement);
-      }
-    });
-  },
-
-  _applyPopper(button, picker) {
-    // eslint-disable-next-line
-    Popper.createPopper(button, picker, {
-      placement: "bottom",
-      modifiers: [
-        {
-          name: "offset",
-          options: {
-            offset: [0, -5]
-          }
-        },
-        {
-          name: "preventOverflow",
-          options: {
-            padding: 5
-          }
-        }
-      ]
-    });
+  didRenderWidget() {
+    if (!this[POPPER_NAME]) {
+      this._setupPopper(POPPER_NAME, ".user-list");
```

Comment:
Can we not set up this popper when in mobileView? as we won't be using it in mobileview

Reply:
yes indeed good call :+1: